### PR TITLE
Document current acl:trustedApp system

### LINKED ISSUE #38: remove "possible future" label from `acl:trustedApp` paragraph
In today's community meeting, we decided to go ahead with the `acl:trustedApp` system, so we should  reword https://github.com/solid/web-access-control-spec#possible-future to say that it's currently part of the spec.

### PR DESCRIPTION
**Pull request by michielbdejong**
Fixes #38

### CODE REVIEW

**Comment by michielbdejong** [comment]
@kjetilk please review? I added the note that it was only added recently and is likely to change in the near future.

**Comment by michielbdejong** [comment]
Thanks! Now it looks like we need a second review by a reviewer with write access, and then someone should merge it.

Not sure who is currently a reviewer on this repo and could help with that - @RubenVerborgh and @dmitrizagidulin probably?

@Mitzi-Laszlo can you also add @jaxoncreed and me on this repo?

**Comment by michielbdejong** [comment]
@megoth maybe?

File: README.md [hunk 1]
```diff
@@ -386,17 +386,15 @@ the origin MUST be allowed access*
...
 
 In solid a maxim is, you have complete control of he data. Therefore it is up to the owner of the data, the publisher, the controller of the ACL, or more broadly the person running the solid server, to specify who gets access, be it people or apps.   However another maxim is that you can chose which app you use.  So of Alice publishes data, and Bob want to use his favorite app,  then how does that happen?  
 
-##### Now:
-
- - The web server can run with a given trusted domain created by the solid developers.
+ - The web server can run with a given trusted domain configured by the pod provider.
```
Comment (RubenVerborgh) [inline README.md]: Not sure about "the web server can run", what does that mean?

Comment (michielbdejong) [reply README.md]: I read that as referring to the option of whitelisting an origin server-wide. I changed the person who can do this from 'the solid developers' to 'the pod provider'. Open for suggestions to rephrase the first part of the sentence too! Maybe 'The pod provider can configure the pod server with a whitelist of origins that are always allowed'

Comment (RubenVerborgh) [reply README.md]: "A Web server MAY be configured such that a given list of origins is unconditionally trusted for incoming HTTP requests. The origin check is then bypassed for these domains, but other access control mechanisms remain in place"

**Comment by michielbdejong** [comment]
@kjetilk @RubenVerborgh can you re-review with that latest edit? ^

**Comment by TallTed** [comment]
"added recently" is ambiguous -- and will surprisingly quickly become inaccurate. I strongly advise including some absolute date detail.

**Pull request merged**